API: The `update` command `target` parameter (#234)
borisbu/OctoRelay: README.md

  - The icon you choose for the button will display the current state.
- By sending GCODE command `@OCTORELAY r#`.
  - Where `#` is relay index from `1` to `8`.
- Or by querying the API (see below)

## OctoRelay API
Relays can be queried and updated through the [OctoRelay API](https://docs.octoprint.org/en/master/api/). Read this documentation to get an API Key. 

### Update Command ###
  - Use path: `/api/plugin/octorelay`.

>   - With JSON payload `{ "pin": "r#", "command": "update", target: "on" }`.

Reviewer: consider operating booleans in JSON

```suggestion
  - With JSON payload `{ "pin": "r#", "command": "update", target: true }`.
```

  - Where `#` is relay index from `1` to `8`.

>   - `target` is an optional parameter. Valid values are `on` or `off`. When this parameter is omitted the relay will toggle.

Reviewer: ```suggestion
  - `target` is an optional boolean parameter. Valid values are `true` or `false`. When this parameter is omitted the relay will toggle.
```

  - The icon you choose for the button will display the current state.
- By sending GCODE command `@OCTORELAY r#`.
  - Where `#` is relay index from `1` to `8`.

> - Or by querying the API (see below)

Reviewer: ```suggestion
- Or by querying the API (see below).
```
